src/exposure.jl

```diff
@@ -0,0 +1,161 @@
+const DEGRADATION_RATE = Param(0.5; bounds=(0.0, 100.0))
+const PULSE_LEVEL = Param(20.0; bounds=(0.0, 100.0))
+const POPULATION_THRESHOLD = Param(20.0; bounds=(0.0, 100.0))
+const CROP_OCCUPANCY = Param(5.0; bounds=(0.0, 10.0))
+const ROTATION_NUMBER = Param(1.0; bounds=(0.0, 1000.0))
+const ROTATION_SIZE = Param(1.0; bounds=(0.0, 1000.0))
+
+"""
+Extends CellRule for rules of exposure dynamics
+
+For best performance these should be chained with other
+CellRule or following an NeighborhoodRule.
+"""
+
+abstract type Exposure{R,W} <: CellRule{R,W} end
+
+"""
+Exposure by treatment depending on a threshold layer (population size) set with `popThreshold`.
+Treatment is applied only on `crop` fields.
+The amount of pesticide `pulseLevel` is applied once this threshold is passed.
+At every time step the pesticide is degradated at rate `degradationRate`
+
+Read: Tuple(pesticide, population)
+Written: Tuple(pesticide)
+"""
+struct Threshold_Exposure{R,W,CP,PL,PT,DR,TS,S} <: Exposure{R,W}
+    "key layer for occurence of plot"
```

Comment:
Can remove these doc strings and instead write them in the main doc string under an `## Arguments` header.

```julia
- `popthreshold`: some description
```